### PR TITLE
Add a template for build cop revert PRs.

### PR DESCRIPTION
**Pull request by david-german-tri**
+@jamiesnape (feature)
+@jwnimmer-tri (platform)

@ggould-tri FYI

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/2403)

### CODE REVIEW

**Comment by jamiesnape** [comment]
:lgtm:

<details><summary><i>Previously, david-german-tri (David German) wrote…</i></summary>
> #### Add a template for build cop revert PRs\.
> 
> +@jamiesnape (feature)
> 
> +@jwnimmer-tri (platform)
> 
> 
> 
> @ggould-tri FYI
> 
> 
> 
</details>


---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-:-KI8cVF1F_lHungpW0Xt:1417088721)*

**Comment by jamiesnape** [comment]
FYI current build cop @jonathan-owens.

**Comment by jwnimmer-tri** [comment]
@david-german-tri BTW you need to post the +@jwnimmer-tri tags from reviewable in order to assign; a github comment doesn't work.

<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> FYI current build cop @jonathan-owens.
</details>


---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-:-KI8eHRBQvnTnK4mWfKT:316950467)*

**Comment by ggould-tri** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[drake/doc/buildcop.rst, line 58 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8e3LK8nYlSszgkZlS:-KI8e3LK8nYlSszgkZlT:-1204757799) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L58)):*
> ```reStructuredText
>  The specific build failures under investigation are:
>  $LINK_TO_JENKINS
>  $LINK_TO_JENKINS
> ```

For clarity, please note that the link should be to the broken build result itself, not to a current or in-progress build page.  This was not the case earlier.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403)*

**Comment by jamiesnape** [comment]
@ggould-tri Ultimately both should be posted on the thread.

**Comment by jamiesnape** [comment]
The in-progress builds are for the reference of the build cop.

**Comment by liangfok** [comment]
I left some small polishing considerations.

<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> The in-progress builds are for the reference of the build cop.
</details>


---

Review status: all files reviewed at latest revision, 9 unresolved discussions.

---

*[drake/doc/buildcop.rst, line 42 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8gJ-288p2EASD6XWn:-KI8gJ-288p2EASD6XWo:-2044406987) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L42)):*
> ```reStructuredText
> pass.
> 
> Revert Template
> ```

Consider adding a header link:

```
.. _revert_template:

Revert Template
```

---

*[drake/doc/buildcop.rst, line 45 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8gW-aMhrdGZsI3ioo:-KI8gW-aMhrdGZsI3iop:-135078744) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L45)):*
> ```reStructuredText
> ---------------
> When creating a revert PR, the build cop will assign that PR to the original
> author, and include the following template in the PR description.
> ```

Consider removing the comma after author.

---

*[drake/doc/buildcop.rst, line 52 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8esrDvI4jJIpSgGTz:-KI8esrDvI4jJIpSgGU-:1503679775) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L52)):*
> ```reStructuredText
> 
>  The oncall build cop, $BUILD_COP, believes that your PR $NUMBER may have broken
>  the Drake continuous integration build. It is possible to break the build even
> ```

Consider: "the Drake" → "Drake's"

---

*[drake/doc/buildcop.rst, line 52 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8geTrEWOcctUSNSzW:-KI8geTrEWOcctUSNSzX:-1042907651) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L52)):*
> ```reStructuredText
> 
>  The oncall build cop, $BUILD_COP, believes that your PR $NUMBER may have broken
>  the Drake continuous integration build. It is possible to break the build even
> ```

Consider "continuous integration build" → "continuous integration build (https://drake-jenkins.csail.mit.edu/view/Continuous/)"

---

*[drake/doc/buildcop.rst, line 61 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fPsHC4FS_akZIhMe:-KI8fPsHC4FS_akZIhMf:-1587817140) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L61)):*
> ```reStructuredText
>  ...
> 
>  Therefore, the build cop has created this revert PR and started a complete
> ```

Consider: "has created" → "created"

---

*[drake/doc/buildcop.rst, line 62 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fSbYNhAyOknGpU7m:-KI8fSbYNhAyOknGpU7n:1413545383) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L62)):*
> ```reStructuredText
> 
>  Therefore, the build cop has created this revert PR and started a complete
>  postsubmit build. If you do nothing, this PR will be merged if that build
> ```

Consider: "build" → "build to determine whether your PR was in fact the cause of the problem."

---

*[drake/doc/buildcop.rst, line 63 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fc6qsEI7JHB-3E0s:-KI8fc6rI4_zRVe1Y2IJ:-1933878067) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L63)):*
> ```reStructuredText
>  Therefore, the build cop has created this revert PR and started a complete
>  postsubmit build. If you do nothing, this PR will be merged if that build
>  passes, no sooner than 60 minutes from now. You can then fix the problem at
> ```

Consider: "now" → "now, assuming the revert fixed the continuous build problem."

---

*[drake/doc/buildcop.rst, line 69 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fn0LPFKKgHCY3N1K:-KI8fn0LPFKKgHCY3N1L:-1215151191) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L69)):*
> ```reStructuredText
>  explain on this thread.
> 
>  If you believe you can fix the break promptly in lieu of revert, please 
> ```

Consider: "of" → "of a"

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-:-KI8h81Offc1PYcqi4Wx:905068352)*

**Comment by jamiesnape** [comment]
Review status: all files reviewed at latest revision, 9 unresolved discussions.

---

*[drake/doc/buildcop.rst, line 45 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8gW-aMhrdGZsI3ioo:-KI8haQB2FewKPPh4muo:1342948505) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L45)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Consider removing the comma after author.
</details>
The comma is correct, as would be a colon.

---

*[drake/doc/buildcop.rst, line 52 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8esrDvI4jJIpSgGTz:-KI8hfCzrfucICXXRHEl:425769076) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L52)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Consider: "the Drake" → "Drake's"
</details>
Either is fine.

---

*[drake/doc/buildcop.rst, line 61 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fPsHC4FS_akZIhMe:-KI8hj0MsxlBPT15mCKD:-1014887148) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L61)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Consider: "has created" → "created"
</details>
Normally you would say "has created".

---

*[drake/doc/buildcop.rst, line 63 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fc6qsEI7JHB-3E0s:-KI8hwAmVuOxJYhV4xJf:528940329) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L63)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Consider: "now" → "now, assuming the revert fixed the continuous build problem."
</details>
continuous integration build?

---

*[drake/doc/buildcop.rst, line 69 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fn0LPFKKgHCY3N1K:-KI8i-pWBObvJbHaZZcG:1775589054) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L69)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Consider: "of" → "of a"
</details>
I agree.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403)*

**Comment by jwnimmer-tri** [comment]
Platform :lgtm:

<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> I left some small polishing considerations.
</details>


---

Reviewed 1 of 2 files at r1.
Review status: all files reviewed at latest revision, 9 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-:-KI9631ovIOrR4TCYu3I:528945684)*

**Comment by liangfok** [comment]
Review status: all files reviewed at latest revision, 9 unresolved discussions.

---

*[drake/doc/buildcop.rst, line 45 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8gW-aMhrdGZsI3ioo:-KI96t8eISNsmBtjOPiP:-869135569) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L45)):*
<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> The comma is correct, as would be a colon.
</details>
I'm less certain but will leave it to the author's discretion. 

---

*[drake/doc/buildcop.rst, line 52 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8esrDvI4jJIpSgGTz:-KI97-fyltd0OMvxj9m7:-1988314255) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L52)):*
<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> Either is fine.
</details>
The fewer the words the better in my opinion.

---

*[drake/doc/buildcop.rst, line 61 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fPsHC4FS_akZIhMe:-KI977LepVyNYrhSoG3s:361496434) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L61)):*
<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> Normally you would say "has created".
</details>
Would the sentence be less clear or incorrect without the "has"? If not, I would vote to remove "has" since the fewer words the better.

---

*[drake/doc/buildcop.rst, line 63 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fc6qsEI7JHB-3E0s:-KI97NM3ex6hIFeOV1cH:1326906498) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L63)):*
<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> continuous integration build?
</details>
Sounds good. Consider: "now" → "now, assuming the revert fixed the continuous integration build problem."

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403)*

**Comment by jamiesnape** [comment]
Review status: all files reviewed at latest revision, 9 unresolved discussions.

---

*[drake/doc/buildcop.rst, line 61 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fPsHC4FS_akZIhMe:-KI9VymD7hcI-ZGfm878:1385778557) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L61)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Would the sentence be less clear or incorrect without the "has"? If not, I would vote to remove "has" since the fewer words the better.
</details>
The tense of the sentence would be different (present perfect simple vs. simple past). The pull request was recently created and is ongoing.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403)*

**Comment by liangfok** [comment]
Review status: all files reviewed at latest revision, 9 unresolved discussions.

---

*[drake/doc/buildcop.rst, line 61 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fPsHC4FS_akZIhMe:-KI9YUvCIDHWZ5F8wpx1:-663872453) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L61)):*
<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> The tense of the sentence would be different (present perfect simple vs. simple past). The pull request was recently created and is ongoing.
</details>
I see. So b/c the creation of the revert PR is recent and ongoing, we should include the "has". Works for me. Thanks. I'll mark this thread as "satisfied".

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403)*

**Comment by jamiesnape** [comment]
Review status: all files reviewed at latest revision, 8 unresolved discussions.

---

*[drake/doc/buildcop.rst, line 45 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8gW-aMhrdGZsI3ioo:-KI9XuLR5A8FJOGWoWpg:584825385) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L45)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> I'm less certain but will leave it to the author's discretion. 
</details>
I commented on totally the wrong line there. I thought that comment was attached to "Dear $AUTHOR" for some reason. The comma would conventionally be omitted, but can be left in for clarity. Hence "author's discretion" sounds appropriate.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403)*

**Comment by jwnimmer-tri** [comment]
(Sorry, I forgot to click the eyeball on both files.  Doing so now...)

<details><summary><i>Previously, jwnimmer-tri (Jeremy Nimmer) wrote…</i></summary>
> Platform :lgtm:
</details>


---

Reviewed 1 of 2 files at r1.
Review status: all files reviewed at latest revision, 9 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-:-KI9ZIWCcNxjB0YjBWun:1949480111)*

**Comment by david-german-tri** [comment]
Review status: all files reviewed at latest revision, 7 unresolved discussions.

---

*[drake/doc/buildcop.rst, line 42 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8gJ-288p2EASD6XWn:-KI9akXJMRlQ-gAtQD4I:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L42)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Consider adding a header link:
> 
> ```
> .. _revert_template:
> 
> Revert Template
> ```
</details>
Done.

---

*[drake/doc/buildcop.rst, line 45 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8gW-aMhrdGZsI3ioo:-KI9c07TxBz-8ag8jd4-:-948904386) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L45)):*
<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> I commented on totally the wrong line there. I thought that comment was attached to "Dear $AUTHOR" for some reason. The comma would conventionally be omitted, but can be left in for clarity. Hence "author's discretion" sounds appropriate.
</details>
Closing.

---

*[drake/doc/buildcop.rst, line 52 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8esrDvI4jJIpSgGTz:-KI9aooKLEh_SxjwJAMc:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L52)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> The fewer the words the better in my opinion.
</details>
Done.

---

*[drake/doc/buildcop.rst, line 52 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8geTrEWOcctUSNSzW:-KI9b5PBJvgLyGkwaXZB:79453838) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L52)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Consider "continuous integration build" → "continuous integration build (https://drake-jenkins.csail.mit.edu/view/Continuous/)"
</details>
Added in the footer.

---

*[drake/doc/buildcop.rst, line 58 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8e3LK8nYlSszgkZlS:-KI9auyHP7uCJNXROcvW:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L58)):*
<details><summary><i>Previously, ggould-tri wrote…</i></summary>
> For clarity, please note that the link should be to the broken build result itself, not to a current or in-progress build page.  This was not the case earlier.
</details>
Done.

---

*[drake/doc/buildcop.rst, line 61 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fPsHC4FS_akZIhMe:-KI9c3OTea_xuYEYG4Lo:-948904386) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L61)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> I see. So b/c the creation of the revert PR is recent and ongoing, we should include the "has". Works for me. Thanks. I'll mark this thread as "satisfied".
</details>
Closing.

---

*[drake/doc/buildcop.rst, line 62 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fSbYNhAyOknGpU7m:-KI9bVFov8TRw5_kbmH_:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L62)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Consider: "build" → "build to determine whether your PR was in fact the cause of the problem."
</details>
Done.

---

*[drake/doc/buildcop.rst, line 63 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fc6qsEI7JHB-3E0s:-KI9bnqQb_3nBXfABohs:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L63)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Sounds good. Consider: "now" → "now, assuming the revert fixed the continuous integration build problem."
</details>
Done.

---

*[drake/doc/buildcop.rst, line 69 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI8fn0LPFKKgHCY3N1K:-KI9bqMFIsGAX7zKYrfS:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/9d201dfb63b44568da03840871bf534b13acdcbf/drake/doc/buildcop.rst#L69)):*
<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> I agree.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403)*

**Comment by jwnimmer-tri** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403)*

**Comment by liangfok** [comment]
Just one more consideration.

<details><summary><i>Previously, jwnimmer-tri (Jeremy Nimmer) wrote…</i></summary>
> (Sorry, I forgot to click the eyeball on both files.  Doing so now...)
</details>


---

Review status: all files reviewed at latest revision, 5 unresolved discussions.

---

*[drake/doc/buildcop.rst, line 58 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI9fMy-KieAbWZTPWRx:-KI9fMy0wLw0TLvbaQ1i:1495129107) ([raw file](https://github.com/robotlocomotion/drake/blob/14309e50a1fc6b00c53049ca1cc64af1c88deb3d/drake/doc/buildcop.rst#L58)):*
> ```reStructuredText
> 
>  The oncall build cop, $BUILD_COP, believes that your PR $NUMBER may have broken
>  Drake's continuous integration build. It is possible to break the build even
> ```

Consider: "build" → "build [1]"

---

*[drake/doc/buildcop.rst, line 85 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI9fSP_reOQJOaJYyFI:-KI9fSP_reOQJOaJYyFJ:-873811471) ([raw file](https://github.com/robotlocomotion/drake/blob/14309e50a1fc6b00c53049ca1cc64af1c88deb3d/drake/doc/buildcop.rst#L85)):*
> ```reStructuredText
>  Your Friendly Oncall Buildcop
> 
>  CI Dashboard: https://drake-jenkins.csail.mit.edu/view/Continuous/
> ```

Consider: "CI" → "[1] CI"

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-:-KI9fYrf7lbGYE-rVFb6:-2056605004)*

**Comment by david-german-tri** [comment]
Review status: 1 of 2 files reviewed at latest revision, 3 unresolved discussions.

---

*[drake/doc/buildcop.rst, line 58 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI9fMy-KieAbWZTPWRx:-KI9o0TFJAwfalk7Co3t:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/14309e50a1fc6b00c53049ca1cc64af1c88deb3d/drake/doc/buildcop.rst#L58)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Consider: "build" → "build [1]"
</details>
Done.

---

*[drake/doc/buildcop.rst, line 85 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2403#-KI9fSP_reOQJOaJYyFI:-KI9o1_qgXcoR4N4kTjL:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/14309e50a1fc6b00c53049ca1cc64af1c88deb3d/drake/doc/buildcop.rst#L85)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Consider: "CI" → "[1] CI"
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2403)*

**Comment by jwnimmer-tri** [comment]
Using override-merge due to out-of-date-with-master and jenkins-not-done-yet, both of don't matter for doc-only edits.

**Comment by jamiesnape** [comment]
> jenkins-not-done-yet, both of don't matter for doc-only edits.
Though, documentation is now built on `linux_matlab` pull request builds, so there is very vague possibility of failure.

**Comment by jwnimmer-tri** [comment]
Ah my mistake.  I thought it was going to be a separate column, so I assumed it wasn't up and running yet.  Wouldn't a separate column be better?  Give clearer failure modes, don't make the matlab builds take any longer, etc?

**Comment by snape** [comment]
It only adds about a minute, which is quicker than the start up time of a new instance. I guess the build of MATLAB documentation does not really need to be tied to the MATLAB build, so I could just move it across to the non-MATLAB build.

**Comment by jwnimmer-tri** [comment]
Let me rephrase: for documentation-only PRs, it would be nice to have the fastest possible pass / fail result from jenkins; waiting for matlab tests (or any tests) to run first seems like it will would things down somewhat.

**Comment by jwnimmer-tri** [comment]
I assume it would help if I just filed this an an issue (feature request), and got it into the queue?  Probably overloading this PR discussion isn't helping anyone.

**Comment by jamiesnape** [comment]
Sure, file an issue, though reporting build failures early is something I am considering.
